internal/configs/sharding/instance.go

```diff
@@ -7,6 +7,18 @@ import (
 	"github.com/harmony-one/harmony/internal/genesis"
 )
 
+// NetworkID is the network type of the blockchain.
+type NetworkID byte
+
+//Consensus and other message categories
```

Comment:
please fix the comment.  Also, please add a String function to the NetworkID type, so that it will be easier for debugging.